core/edit/edit.py

```diff
@@ -3,6 +3,41 @@
 ctx = Context()
 mod = Module()
 
+SYMBOLS_WHICH_SIGNIFY_THE_END_OF_A_WORD = [
+    ".",
+    "!",
+    "?",
+    ";",
+    ":",
+    "—",
+    "_",
+    "/",
+    "\\",
+    "|",
+    "@",
+    "#",
+    "$",
+    "%",
+    "^",
+    "&",
+    "*",
+    "(",
+    ")",
+    "[",
+    "]",
+    "{",
+    "}",
+    "<",
+    ">",
+    "=",
+    "+",
+    "-",
+    "~",
+    "`",
+    " ",
+    "",
+]
```

Comment:
Quotation marks are left out of this list - they didn't seem to cause issues in my brief testing of them.